nextjs-start/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["eslint:recommended", "next", "prettier"],
+  "rules": {
+    "@next/next/no-img-element": "off",
+    "no-unused-vars": "off"
```

Comment:
Is this because there are placeholder uninitialized variables in the starter code?

Reply:
yep